lib/ret/discord_client.ex

```diff
@@ -76,12 +76,20 @@ defmodule Ret.DiscordClient do
         {status, result} when status in [:commit, :ok] -> "#{result["nick"]}"
       end
 
-    if nickname == "" do
+    nickname = if nickname == "" do
```

Comment:
The cautious check would be `if !nickname or nickname == "" do` but I'm fine either way.

Reply:
I like cautious :)  Good catch, thanks.  Updated.